cf-agent/verify_files.c

```diff
@@ -542,9 +559,26 @@ static PromiseResult VerifyFilePromise(EvalContext *ctx, char *path, const Promi
 
     if (a.haveedit)
     {
-        if (exists)
+        /* Files promises that promise full file content - like the template
+         * methods 'mustache' and 'inline_mustache' - are exeptions to the
```

Comment:
One would expect the `content` attribute mentioned here and handled in the same way.

Reply:
Content attribute currently ignores `create => "false"`, thus it should be fixed in another ticket first.

Reply:
Ticket: [CFE-3916](https://tracker.mender.io/browse/CFE-3916)